src/layers/sds/cpm_translation/modify_key.py

```diff
@@ -136,30 +147,37 @@ def replace_msg_handling_system(
     device.delete()
```

Comment:
add a ticket as tech debt so that you can't replace an mhs with itself (otherwise would have a key clash - should fail?)